schemas/context/graph-edge.schema.json

```diff
@@ -0,0 +1,127 @@
+{
+    "meta:license": [
+      "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+      "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+      "you may not use this file except in compliance with the License. You may obtain a copy",
+      "of the License at https://creativecommons.org/licenses/by/4.0/"
+    ],
+    "$id": "https://ns.adobe.com/xdm/context/graph-edge",
+	"$schema": "http://json-schema.org/draft-06/schema#",
+	"title": "XDM Graph Edge",
+	"type": "object",
+	"description": "XDM Graph Schema for Edge/Relationship between two nodes/vertices.",
+	"auditable": true,
+	"meta:extensible": true,
+	"meta:abstract": true,
+	"meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
```

Comment:
Is this correct, that these objects are representations of a Profile? 

Or, should this become it's own type of class?

Reply:
@cmathis , Initially the intent was to have the graphs exist as its own class and have the Node and Edge mixins extend this, we later made this extend profile class assuming that it would be used along with the profile data. We will get more clarity once we have feedback from the Identity services team.